test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
I added a ``test_package`` that runs automatically when ``conan create .`` to do a quick validation of the package.